server/src/postgis/flight.rs

```diff
@@ -346,11 +350,207 @@ pub async fn get_flight_intersection_stmt(
                 "(get_flight_intersection_stmt) could not prepare cached statement: {}",
                 e
             );
-            Err(PostgisError::FlightPath(FlightPathError::DBError))
+            Err(PostgisError::FlightPath(FlightError::DBError))
         }
     }
 }
 
+/// Get flights and their aircraft that intersect with the provided geometry
+///  and time range.
+pub async fn get_flights(request: GetFlightsRequest) -> Result<Vec<Flight>, FlightError> {
+    postgis_debug!("(get_flights) entry.");
+
+    let Some(time_start) = request.time_start else {
+        postgis_error!("(get_flights) time_start is required.");
+        return Err(FlightError::Time);
+    };
+
+    let Some(time_end) = request.time_end else {
+        postgis_error!("(get_flights) time_end is required.");
+        return Err(FlightError::Time);
+    };
+
+    let time_start: DateTime<Utc> = time_start.into();
+    let time_end: DateTime<Utc> = time_end.into();
+    let linestring = LineStringT {
+        points: vec![
+            Point {
+                x: request.window_min_x,
+                y: request.window_min_y,
+                srid: Some(DEFAULT_SRID),
+            },
+            Point {
+                x: request.window_max_x,
+                y: request.window_max_y,
+                srid: Some(DEFAULT_SRID),
+            },
+        ],
+        srid: Some(DEFAULT_SRID),
+    };
+
+    let Some(pool) = crate::postgis::DEADPOOL_POSTGIS.get() else {
+        postgis_error!("(get_flights) could not get psql pool.");
+
+        return Err(FlightError::Client);
+    };
+
+    let client = pool.get().await.map_err(|e| {
+        postgis_error!(
+            "(get_flights) could not get client from psql connection pool: {}",
+            e
+        );
+        FlightError::Client
+    })?;
+
+    let stmt = client
+        .prepare_cached(&format!(
+            r#"
+            SELECT 
```

Comment:
Query is missing table name quotes. Especially the ones on the variable names

Reply:
https://github.com/Arrow-air/svc-gis/pull/19/commits/208ea9f19d9c58fd9d971792242d17f2a1b0b87e